pkg/nuctl/test/function_test.go

```diff
@@ -165,11 +201,17 @@ func (suite *functionDeployTestSuite) TestDeployWithMetadata() {
 
 func (suite *functionDeployTestSuite) TestDeployFromFunctionConfig() {
 	randomString := xid.New().String()
-	uniqueSuffix := "-" + randomString
-	imageName := "nuclio/deploy-test" + uniqueSuffix
-	functionName := "parser"
 
-	err := suite.ExecuteNuctl([]string{"deploy", "", "--verbose", "--no-pull"},
+	functionPath := path.Join(suite.GetFunctionsDir(), "common", "json-parser-with-function-config", "python")
+	functionConfig := functionconfig.Config{}
+	functionBody, err := ioutil.ReadFile(filepath.Join(functionPath, build.FunctionConfigFileName))
+	suite.Require().NoError(err)
+	err = yaml.Unmarshal(functionBody, &functionConfig)
+	suite.Require().NoError(err)
+	functionName := functionConfig.Meta.Name
```

Comment:
No need for unique suffix here?

Reply:
I will leave it for next pr, it is getting bigger and a better suite is required for that